IPs/RSKIP170.md

```diff
@@ -70,17 +70,46 @@ When a BTC transaction is sent to the Bridge, the following steps take place:
 - There is no limit to the amount of outputs with OP_RETURN the peg-in transaction can have. But there can be only one with `RSKT` prefix indicating peg-in instructions. Transactions with more than one OP_RETURN outputs containing `RSKT` prefix will be rejected, following the existing protocol for refunds.
 - The output with OP_RETURN containing the peg-in instructions does not have to be in any particular order among the other outputs.
 
+### New event in bridge `pegin_btc`
+Currently, when a peg-in is completed succesfully the bridge emits an event called `lock_btc` with the following information:
+- [Topic] RSK adddress that received the funds
+- Hash of the BTC transaction that generated the peg-in
+- String representation of the sender BTC address, e.g. base58, bech32
+- The amount (in weis) transferred to the recipient
+
+Signature:
+
+`lock_btc(address indexed receiver, bytes32 btcTxHash, string senderBtcAddress, int256 amount)`
```

Comment:
```suggestion
`pegin_btc(address indexed receiver, bytes32 btcTxHash, string senderBtcAddress, int256 amount)`
```

Reply:
It's the signature of the "old" event lock_btc. The signature for the new event is below